packages/cli/src/databases/migrations/common/1740445074052-UpdateParentFolderIdColumn.ts

```diff
@@ -0,0 +1,97 @@
+import type { BaseMigration, MigrationContext } from '@/databases/types';
+
+export class UpdateParentFolderIdColumn1740445074052 implements BaseMigration {
+	async up({
+		escape,
+		queryRunner,
+		copyTable,
+		schemaBuilder: { dropForeignKey, createTable, column },
+		isPostgres,
+		isMysql,
+		tablePrefix,
+	}: MigrationContext) {
+		const workflowTableName = escape.tableName('workflow_entity');
+		const folderTableName = escape.tableName('folder');
+		const parentFolderIdColumn = escape.columnName('parentFolderId');
+		const idColumn = escape.columnName('id');
+
+		if (isMysql) {
+			// Initial migration did not add the foreign key constraint so no need for the delete operation
+			await queryRunner.query(
+				`ALTER TABLE ${workflowTableName} ADD CONSTRAINT fk_workflow_parent_folder FOREIGN KEY (${parentFolderIdColumn}) REFERENCES ${folderTableName}(${idColumn}) ON DELETE CASCADE`,
+			);
+		} else if (isPostgres) {
+			const workflowTable = await queryRunner.getTable(`${tablePrefix}workflow_entity`);
```

Comment:
Is concatenating `tablePrefix` to the table name without escaping the resulting string safe in these migrations, could the prefix contain uppercase letters for instance?